book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] A Book has many rentals, so you need to add `add_rental` method, for more info refer to the previous lessons for [associations](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship).
> It will be like this:
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```